pkg/container-disk/container-disk.go

```diff
@@ -160,11 +162,20 @@ func GenerateContainers(vmi *v1.VirtualMachineInstance, podVolumeName string, bi
 
 	// Make VirtualMachineInstance Image Wrapper Containers
 	for index, volume := range vmi.Spec.Volumes {
+
```

Comment:
The pattern of assigning a containerdisk vad based on the volume type (containerdisk or copyonwrite) appears 6 times. The code should be extracted in a function (something like GetReferencedContainerDisk). The same could be done for the referenced pvc for symmetry.

Reply:
yep, great idea. i'll make sure to clean this up